includes/bootstrap.php

```diff
@@ -21,3 +21,17 @@
 		include $file_to_include;
 	}
 }
+
+// Add back-compat aliases for amp-toolbox-php interfaces which were renamed in v0.9.0.
+spl_autoload_register(
+	function ( $class_name ) {
+		switch ( $class_name ) {
+			case 'AmpProject\Attribute':
+				class_alias( \AmpProject\Html\Attribute::class, $class_name );
+				break;
+			case 'AmpProject\Tag':
+				class_alias( \AmpProject\Html\Tag::class, $class_name );
+				break;
+		}
+	}
+);
```

Comment:
This will be reverted once amp-toolbox-php 0.9.2 is released.

Reply:
Related PR: https://github.com/ampproject/amp-toolbox-php/pull/434